data/scenarios/Tutorials/build.yaml

```diff
@@ -15,8 +15,8 @@ objectives:
         where in place of `COMMANDS`{=snippet} you write the sequence
         of commands for the robot to execute (separated by semicolons).
       - |
-        Build a robot to harvest the `"flower"` and place it next
-        to the water.
+        Build a robot to harvest the `flower`{=entity} and place it next
```

Comment:
@kostmo I mentioned this in the other PR, but what do you think about showing the quotes to the player because that is what they need to write to REPL? 🤔

Reply:
The raw `yaml` markdown should not require quotes, but yes, perhaps additional double quotes could be rendered in-game.

Reply:
I don't think we should *always* render entity names with double quotes, that would he annoying.  I think there are just a few places like this in the tutorials where it might be helpful.